config/sites/classrooms.uiowa.edu/field.field.taxonomy_term.technology_features.field_api_mapping.yml

```
@@ -0,0 +1,19 @@
+uuid: 630a5aef-0e34-4fd1-990a-4eb4b877d7c1
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.taxonomy_term.field_api_mapping
+    - taxonomy.vocabulary.technology_features
+id: taxonomy_term.technology_features.field_api_mapping
+field_name: field_api_mapping
+entity_type: taxonomy_term
+bundle: technology_features
+label: 'API Mapping'
+description: 'Enter the MAUI featureList name of the feature to add, for example Computer - Windows.'
```

```suggestion
description: 'Enter value from the MAUI <em>featureList</em> property to map this term to (e.g. <em>Computer - MAC</em>).'
```
This is a good example to emphasize the case sensitivity of the mapping.